pkg/rebuild/debian/strategy.go

```diff
@@ -81,13 +81,24 @@ func (b *Debrebuild) ToWorkflow() *rebuild.WorkflowStrategy {
 				"buildinfoMd5": b.BuildInfo.MD5,
 			},
 		}},
-		Build: []flow.Step{{
-			Uses: "debian/build/debrebuild",
+		Deps: []flow.Step{{
+			Uses: "debian/deps/install",
 			With: map[string]string{
-				"targetPath": "{{.Target.Artifact}}",
-				"buildinfo":  path.Base(b.BuildInfo.URL),
+				"requirements": flow.MustToJSON([]string{"devscripts", "apt-utils", "mmdebstrap"}),
```

Comment:
I think you could also add these as system deps (`Needs`) of the debrebuild tool which would mean you could drop this step.